index.ts

```diff
@@ -1,4 +1,20 @@
-import { Configs, KlasaClient } from './imports';
+import * as dotenv from 'dotenv';
+import { KlasaClient } from './imports';
+
+dotenv.config();
+let path;
+switch (process.env.NODE_ENV) {
+  case 'test':
+    path = `${__dirname}/../../.env.test`;
+    break;
+  case 'production':
+    path = `${__dirname}/../../.env.production`;
+    break;
+  default:
+    path = `${__dirname}/../../.env.development`;
+}
+dotenv.config({ path });
+export const token = process.env.token;
```

Comment:
Why is this exported?

Reply:
This is if we we're exporting this as a constant, we don't need this, but if we export all of the environment variables, we can just import and use them in other files like a password, and bot support author id's

Reply:
also, if you dislike using process.env a lot, this loses the process and use them as a const, token.